src/t8_forest/t8_forest_iterate.h

```diff
@@ -84,7 +84,7 @@ T8_EXTERN_C_BEGIN ();
 
 /* TODO: Document */
 void
-t8_forest_split_array (const t8_element_t *element, t8_element_array_t *leaf_elements, size_t *offsets);
+t8_forest_split_array (const t8_element_t *element, const t8_element_array_t *leaf_elements, size_t *offsets);
```

Comment:
Since you are on it, can you document the function, too?

Can the offsets be const, too?

Reply:
the offsets are output, so no

Reply:
commented the function